app/src/main/java/com/rockthevote/grommet/ui/eventFlow/PartnerLoginViewModel.kt

```diff
@@ -56,6 +58,7 @@ class PartnerLoginViewModel(
                     }
                 }.onSuccess {
                     partnerInfoDao.deleteAllPartnerInfo()
+                    sessionDao.clearAllSessionInfo()
```

Comment:
GROM 148 and 163 are both in this PR, right?

Reply:
Yeah